File: examples/python/basic_lookup.py
from bd_offline_geocoder import BdOfflineGeocoder


def main() -> None:
    geocoder = BdOfflineGeocoder.from_included_data()
    result = geocoder.reverse(latitude=23.74015, longitude=90.38286)

    print(result.formatted)
    print(result.to_dict())


if __name__ == "__main__":
    main()

File: examples/npm/basic-lookup.js
import { BdOfflineGeocoder } from "../../packages/npm/src/index.js";

const geocoder = BdOfflineGeocoder.fromIncludedData();
const result = geocoder.reverse({ latitude: 23.74015, longitude: 90.38286 });

console.log(result.formatted);
console.log(JSON.stringify(result.toJSON(), null, 2));
